dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -171,8 +171,8 @@ customClaims.desc=The extra claims to be put in the payloads of the ID token, in
 thirdPartyIDTokenClaims=Extra claims from a third-party ID token
 thirdPartyIDTokenClaims.desc=The extra claims from a third-party ID token to be put in the payload of the ID token.
 
-jwkEnabled=Enable JSON web key(JWK)
-jwkEnabled.desc=Enables or disables JWK.
+jwkEnabled=Enable JSON web key (JWK)
+jwkEnabled.desc=When JWK is enabled, the OpenID Connect provider dynamically generates key pairs for signing JWT tokens that it creates. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://<host_name>:<port_number>/oidc/endpoint/<provider_id>/jwk. When this attribute is false, the provider will use the key that is specified by the keyAliasName attribute to sign the JWT. The provider will still make that key available as a JWK from the JWK endpoint if this attribute is false.
```

Comment:
Proposed text:
"jwkEnabled.desc=When JWK is enabled, the OpenID Connect provider dynamically generates key pairs for signing JWT tokens that it creates. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://<host_name>:<port_number>/oidc/endpoint/<provider_id>/jwk. When this attribute is false, the client can still retrieve the key that is specified by the keyAliasName attribute as a JWK from the JWK endpoint."

Reply:
@hiltoncv I don't see any difference between your proposed text and what I have.

Reply:
With the latest iteration, the proposed text is not different from the original text. I will change the label to ID review not required.

Reply:
> Proposed text:
> "jwkEnabled.desc=When JWK is enabled, the OpenID Connect provider dynamically generates key pairs for signing JWT tokens that it creates. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://<host_name>:<port_number>/oidc/endpoint/<provider_id>/jwk.
> When this attribute is false, the provider will use the key that is specified by the keyAliasName attribute to sign the JWT. The provider will still make that key available as a JWK from the JWK endpoint if this attribute is false."

Disregard the proposed text. The original text can be retained.